### DISCUSSION TITLE
Invoice for tips

### DISCUSSION THREAD

**Discussion by OriginalTexasToast** [btcpayserver/btcpayserver#4159] [Technical Support]
I am setting up BTCPay server for my restaurant.  In the US tips are taxed differently than other payments, and we need documentation.  Is there a way to see the tip as a subset of the invoice from the BTCPay Server dashboard or invoices?

**Comment by dennisreimann** [comment]
They are accounted for separately (at least if you use the cart view), but right now we aren't displaying the details in the receipt, just in the invoice details.

![grafik](https://user-images.githubusercontent.com/886/192100075-919362e9-5321-43d7-8b63-3419df5f77e6.png)

We can improve this once we get to #4114.

**Reply by OriginalTexasToast** [reply]
For my restaurant, I've found it most convenient to use the POS app.  When I receive a payment from POS I can't find an invoice that includes a tip amount. Am I missing something?

**Reply by bolatovumar** [reply]
@OriginalTexasToast do you not get this section on the invoice details page?

![Capture](https://user-images.githubusercontent.com/1934678/194220466-9023a969-8d41-4fba-b8f3-c89cefb77f66.PNG)

**Reply by OriginalTexasToast** [reply]
No. It looks like that invoice is from a cart.  I am currently using the POS app with Keypad only.  This seems to be the best way to accept btc at my restaurant, but without having the tip amount clearly visible in the invoice details il be forced to go through an extra step of trying to record the tip amount before the customer sends the payment :|

**Reply by dennisreimann** [reply]
We'll fix that once we get to #4114 after v1.7
